docs/API.md

```diff
@@ -530,7 +530,7 @@ if err != nil {
 ```
 
 <a name="FPutObject"></a>
-### FPutObject(bucketName, objectName, filePath, contentType string) error
+### FPutObject(bucketName, objectName, filePath, contentType string) (length int, err error)
```

Comment:
Actually it is int64 not int.

Can you correct it before merging it ? Thanks anyway.